provider/resources.go

```diff
@@ -1759,6 +1777,9 @@ func Provider() tfbridge.ProviderInfo {
 					Source: "datasource_compute_global_address.html.markdown",
 				},
 			},
+			"google_compute_global_forwarding_rule": {
+				Tok: gcpDataSource(gcpComposer, "getGlobalForwardingRule"),
```

Comment:
Should it be `gcpCompute`?
```suggestion
				Tok: gcpDataSource(gcpCompute, "getGlobalForwardingRule"),
```

Reply:
great catch!